jkube-kit/common-maven/src/main/java/org/eclipse/jkube/kit/common/util/AnsiLogger.java

```diff
@@ -45,257 +45,260 @@
  * @since 31.03.14
  */
 public class AnsiLogger implements KitLogger {
-    // prefix used for console output
```

Comment:
It is quite hard to spot the differences with the code format

Reply:
Yup, I saw that later. The only difference is the usage of the `withFallback` method for logging. This allows to catch **any** exception when using Ansi to log messages.